Core/include/Acts/Seeding/PathSeeder.hpp

```diff
@@ -0,0 +1,185 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Seeding/ISourceLinkGrid.hpp"
+#include "Acts/Surfaces/Surface.hpp"
+#include "Acts/Utilities/Delegate.hpp"
+
+namespace Acts {
+
+/// @brief Seeding algorigthm that extracts
+/// the IP parameters and sorts the source links
+/// into possible track candidates
```

Comment:
Can you explain a bit more how the sorting is done?

Reply:
Done